docroot/modules/custom/va_gov_banner/src/Plugin/Validation/Constraint/RequireScopeValidator.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\va_gov_banner\Plugin\Validation\Constraint;
+
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\va_gov_user\Service\UserPermsService;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Validates the RequireScope constraint.
+ */
+class RequireScopeValidator extends ConstraintValidator implements ContainerInjectionInterface {
+  /**
+   * The User Perms service.
+   *
+   * @var \Drupal\va_gov_user\Service\UserPermsService
+   */
+  protected $userPermsService;
+
+  /**
+   * Constructs the ConstraintValidator object.
+   *
+   * @param \Drupal\va_gov_user\Service\UserPermsService $user_perms_service
+   *   The user perms service.
+   */
+  public function __construct(UserPermsService $user_perms_service) {
+    $this->userPermsService = $user_perms_service;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('va_gov_user.user_perms')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validate($item, Constraint $constraint) {
+    /** @var \Drupal\va_gov_banner\Plugin\Validation\Constraint\RequireScope $constraint */
+    if ($item->getEntity()->get('moderation_state')->getString() !== 'published') {
+      return;
+    }
+    if (empty($item->getEntity()->get('field_target_paths')->getString())) {
```

Comment:
This QA step is failing.
>> add a single space in the path field
save the node as published
 Validate that the node does NOT save and you are told why.


```suggestion
    $path = trim($item->getEntity()->get('field_target_paths')->getString());
    if (empty($path)) {
```